osx/Podfile.lock

```diff
@@ -91,15 +91,15 @@ SPEC CHECKSUMS:
   GHKeychain: 008cdb8bbab5b0dc1da3d0f06a67ce5203231e44
   GHKit: 2a20b51df9ce8f4355b2af2821805d369d610546
   GHODictionary: 225e042596a4b3d5b66b2b24250086b2f95514af
-  KBKit: e84eb021c4db47806e8e0a9379aa150f9bf2f9ac
+  KBKit: 9f139696e2cd916f05ee603e33aabfd558ac8ccd
```

Comment:
Any idea why this changed?

Reply:
The build script yelled at me that I needed to run `pod install`.  But I was on cocoapods 1.4-something (less that the previous version that built `Podfile.lock`, so I did `gem install cocoapods` and it gave me 1.6.1.  Then I did `pod install`, and all three of the things you had questions about changed.  Even if you run this on master, without my changes, it updates these two hashes, so I guess `pod install` just hasn't been run / checked in for a while.

Reply:
Let me verify that, thanks for the explanation....

Reply:
Ok, I saw the same behavior when I reinstalled cocoapods.  Looks like 1.6.1 just came out since I recently installed cocoapods for this build about 1 month ago